snapshot/snapshot.go

```diff
@@ -343,7 +343,11 @@ func (o *snapshotter) Mounts(ctx context.Context, key string) ([]mount.Mount, er
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to get active mount")
 	}
-	return o.mounts(ctx, s, key)
+	mounts, err := o.mounts(ctx, s, key)
+	if err == nil {
```

Comment:
We log  "[prepared remote snapshot](https://github.com/awslabs/soci-snapshotter/blob/cc303960c39d34d53066f8908495a116b046a25e/snapshot/snapshot.go#L288)" when Mount is successful. I don't think this new line of log is necessary. I suggest amending the existing log message to ``SOCI mount successfully returned. Prepared remote snapshot``.

Reply:
I may have misinterpreted what the issue might of have been getting at. Since currently successful remote snapshot mounts are logged, I thought that we may have needed to log if the snapshotter returns a successful set of mounts when running/creating a container.

Reply:
Personally I don't think this PR is really needed. However I think it would be nice to amend the existing log msg to clearly state that mount is successful like I suggested above.